api/app/Http/Middleware/AuditQueryMiddleware.php

```diff
@@ -28,11 +28,11 @@ public function __construct(LoggerInterface $logger)
     public function handle(Request $request, Closure $next)
     {
         $user = $request->user();
+        $referer = request()->headers->get('referer');
         if (! is_null($user) && $user->hasRole('platform_admin')) {
-            $message = 'GraphQL request from platform admin user ['.$user['email'].']';
+            $message = 'Request from platform admin, '.$user['email'].', referer: '.$referer.',';
```

Comment:
Could you remove the "referer:" string in the message?  We'll be able to find this message part by position.
```suggestion
            $message = 'Request from platform admin, '.$user['email'].', '.$referer.',';
```